lib/main.dart

```diff
@@ -85,6 +86,9 @@ Future<void> main(List<String> rawArgs) async {
 
   MediaKit.ensureInitialized();
 
+  // force High Refresh Rate on some Android devices (like One Plus)
+  await FlutterDisplayMode.setHighRefreshRate();
```

Comment:
```suggestion
if (DesktopTools.platform.isAndroid) {
    await FlutterDisplayMode.setHighRefreshRate();
}
```

Need to check android platform to avoid `MissingPlatformImplementation` exception in other platforms

Reply:
Oh my bad, should probably catch PlatformException to be super safe